go/teams/chain.go

```diff
@@ -552,7 +552,7 @@ func (t *TeamSigChainState) IsInviteObsolete(id keybase1.TeamInviteID) bool {
 	return ok
 }
 
-// FindActiveKeybaseInvite finds and returns *first* Keybase-type
+// FindActiveKeybaseInvite finds and returns a Keybase-type
```

Comment:
thanks. the intention was good but I might have made it more confusing rather than less